extendr-api/src/rmacros.rs

```diff
@@ -107,3 +107,53 @@ macro_rules! data_frame {
         lang!("data.frame", $($rest)*).eval_blind()
     };
 }
+
+/// Print via the R output stream.
+///
+/// Works like print! but integrates with R and respects
+/// redirection with functions like sink and capture.output
```

Comment:
Tidyverse style puts parenthese after function names (`sink()`, `capture.output()`) and also puts them in backticks.

Reply:
Oh, and similarly for `print!()` I think.